analyzer/codechecker_analyzer/analyzers/clangsa/result_handler.py

```diff
@@ -29,4 +29,4 @@ def postprocess_result(self):
         context insensitive if it is enabled during analysis.
         """
         if self.report_hash_type == 'context-free':
-            report.use_context_free_hashes(self.analyzer_result_file)
+            use_report_hash(self.analyzer_result_file, version='v2')
```

Comment:
Probably naming it `context-free` instead of `v2` would make the code more readable.
As @Xazax-hun mentioned for the documentation but I think it would be better to rename them in the code too.